docs/development/contribute.md

```diff
@@ -1,8 +1,149 @@
 sidebar_label: Contribute
 slug: /development/contribute
 ---
 
-Contribute page
+We are very happy about your interest in helping Sumé. Our community is made up of people like you, who wish to make a positive impact on society.
+
+Feel welcome, we are grateful for any help you can give us, but first, read our guide to learn how to ask questions and contribute.
+
+- [Code of Conduct](/docs/conduct)
+- [Reporting an Issue](/docs/development/contribute#reporting-an-issue)
+- [Project Management](/docs/development/contribute#project-management-maintainers-only)
+- [Your First Contribution](/docs/development/contribute#your-first-contribution)
+- [Contributing](/docs/development/contribute#contributing)
+- [Get in Touch](/docs/development/contribute#get-in-touch)
+
+## Reporting an Issue
+
+Important: **DO NOT report security vulnerabilities on public issues.** Please report the problem directly to the Sumé team. We will assess the issue as soon as possible based on the best efforts and provide an estimate of when we have a correction and release available for possible public disclosure.
+
+If you need technical help or want to discuss a technical or non-technical topic, you can open an issue for discussion or go directly to the community chat on [discord](https://discord.com/invite/aaJzjBunTZ).
+
+All repositories have templates indicating the type and what information is needed to open an issue.
+
+### Bug reports
+
+If you think you've found a bug:
+
+- **Use the GitHub issue search** - check if the issue has already been reported.
+- **Check if the issue has been fixed** - try to reproduce it using the latest main branch in the repository.
+- **Isolate the problem** - ideally create a reduced test case.
+
+Please try to be as detailed as possible in your report.
+
+### Feature requests
+
+Feature requests are welcome. We are open to new ideas and proposed changes to comply with the user needs.
+
+If you want to contribute, open an issue in the respective repository on [GitHub](https://github.com/sumelms), describe as much as possible what you would like to see in Sumé, why you think it would be necessary, and how it should work.
+
+## Project Management (maintainers only)
+
+In order to keep things organized all our repositories is able to have their own issues, but you can found our [project board with all the tickets here](https://github.com/orgs/sumelms/projects/1).
+
+### Ticket classification
+
+#### Epics and User Stories
+
+- **Epics** are issues that to be solved need to be split into small tasks called *user stories*.
+- **User Stories** are issues that to be solved need to be split into small and executable issues. 
+
+This type of issues we call them "planning issues", 'cause they are the ones that can control and track the execution of the other issues.
+
+These tasks should, in fact, hold the necessary amount of the strategic view of its goal as well the links to other documents and related issues - nothing more, nothing less.
```

Comment:
These tasks should hold the necessary amount of the strategic view of its goal as well the links to other documents and related issues - nothing more, nothing less.

Reply:
Done!